app/views/content/life-as-a-teacher/teaching-as-a-career/change-careers/_article.html.erb

```diff
@@ -0,0 +1,111 @@
+      text: "<p>I’d been working in the same role for many years and I was no longer feeling challenged. So I had a decision to make: stay in my finance role, or give myself a new break.</p><a href='/life-as-a-teacher/teaching-as-a-career/making-a-career-change-to-teaching'>Read more about Peter's career change to teaching</a>",
+      quotes: true,
+      background_color: "blue",
+      heading_color: "pink",
+    ) %>
+  </div>
+</div>
+<div class="row inset">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll need to become a teacher</h2>
+    <p>You’ll need a degree and <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a> to work in the majority of schools in England. This  includes state maintained primary, secondary and special schools.</p>
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to get QTS.</p>
+    <p>It doesn't matter when you graduated, and your degree subject doesn't necessarily need to be the subject you want to teach. Once you have QTS you could teach various subjects, providing you can show you have enough expertise. For example, through your previous career.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-have-a-degree"> Find out more about postgraduate teacher training</a>.</p>
+    <h3>If you do not have a degree</h3>
+    <p>If you do not have a degree, you can:</p>
+    <ul>
+      <li>do undergraduate teacher training to get a degree alongside qualified teacher status QTS</li>
+      <li>check if you're eligible to do a teaching degree apprenticeship</li>
+    </ul>
+    <p>Find out more about <a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">training to teach if you do not have a degree</a>.</p>
+    <h3>If you're unable to get a degree</h3>
+    <p>You do not need a degree and QTS to <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teach in further education</a> or to <a href="/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers">teach in early years</a>. So if you want to become a teacher, but are not able to get a degree, you can explore these options.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I would advise anyone thinking about changing careers that you will be embraced and more importantly an asset to the profession and students.",
+    name: "Ian, career changer",
+    large: false
+  ) %>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Fund your training</h2>
+    <p>Tax-free bursaries and scholarships of upto <%= v :scholarships_generic_maxshortened %> are available for eligible trainee teachers. You can get these alongside a tuition fee loan and maintenance loan.</p>
+    <p>You may also be able to get extra funding and support if you're:</p>
```

Comment:
Could use the partial created for subject pages here?

Reply:
Content is slightly different to subject pages, as we've streamlined the hyperlinks

Reply:
I wonder if at some point, we should do a further review of content and create more partials that are variations - so that if we need to do updates on funding in future, all the partials are in the same place - rather than trying to remember which pages we talk about funding?

Something to discuss with @Sarah-DfE (not for this PR!) - resolve this comment when you've read it!